src/main/java/bowling/views/ResultView.java

```diff
@@ -0,0 +1,127 @@
+        int leftPadding = FRAME_WIDTH - result.length() - rightPadding;
+        return SEPARATOR +
+                StringUtils.generateBlank(leftPadding) +
+                result +
+                StringUtils.generateBlank(rightPadding);
+    }
+
+    private static String getBlankFrameStates() {
+        StringBuilder result = new StringBuilder();
+        for (int i = ResultView.FIRST_ROUND; i < FINAL_ROUND; i++) {
+            result.append(getStateOrName(BLANK));
+        }
+        return result.toString();
+    }
+
+    private static String getBlankOrFrameState(Frame frame) {
+        if (frame.hasResult()) {
+            return getFrameStates(frame.states(), frame.downPins());
+        }
+        return getStateOrName(BLANK);
+    }
+
+    private static String getFrameStates(List<State> states, DownPins downPins) {
+        if (states.size() == FIRST_STATE) {
+            return getFrameState(states.get(FIRST_INDEX), downPins);
+        }
+        String value = getFinalFrameStates(states, downPins);
+        return getStateOrName(value);
+    }
+
+    private static String getFinalFrameStates(List<State> states, DownPins downPins) {
+        return IntStream.range(FIRST_INDEX, states.size())
+                .mapToObj(i -> {
+                    if (State.STRIKE.equals(states.get(i))) {
+                        return STRIKE;
+                    }
+                    return getSpareOrGutter(downPins.get(i).count());
+                })
+                .collect(Collectors.joining(SEPARATOR));
+    }
+
+    private static String getFrameState(State state, DownPins downPins) {
+        if (State.STRIKE.equals(state)) {
+            return getStateOrName(STRIKE);
+        }
+        if (State.SPARE.equals(state)) {
+            String value = getSpareOrGutter(downPins.get(FIRST_INDEX).count()) + SEPARATOR + SPARE;
+            return getStateOrName(value);
+        }
+        return getStateOrName(pinResult(downPins));
+    }
+
+    private static String pinResult(DownPins downPins) {
+        return downPins.downPins()
+                .stream()
+                .map(DownPin::count)
+                .map(ResultView::getSpareOrGutter)
+                .collect(Collectors.joining(SEPARATOR));
+    }
+
+    private static String getSpareOrGutter(int pinCount) {
+        if (pinCount == GUTTER_PIN_COUNT) {
+            return GUTTER;
+        }
+        return String.valueOf(pinCount);
+    }
+
+    public static void showFrameResult(Player player, Frames frames) {
+        StringBuilder stringBuilder = new StringBuilder();
+        stringBuilder.append(getStateOrName(player.name()));
+        for (Frame frame: frames.frames()) {
+            stringBuilder.append(getBlankOrFrameState(frame));
+        }
+        System.out.println(stringBuilder + SEPARATOR + ENTER);
+    }
+
+    public static void showScoreBoard(Player player, Frames frames) {
+        showScoreBoardTitle();
+        showFrameResult(player, frames);
+    }
```

Comment:
```suggestion
    public static void showScoreBoard(Player player, Frames frames) {
        System.out.println(scoreBoardTitle());
        System.out.println(frameResult(player, frames));
    }
```

사소한 피드백이기는 하지만, showScoreBoardTitle 와 showFrameResult 가 String 값을 리턴하게 하고,
System.out 을 하는 것은 showScoreBoard 로 한정짓는 것은 어떠신가요?
취향차이이기는 하지만, 저에게 입출력은 언제나 병목의 위험성이 있기에,
입출력이 일어나는 로직은 추적하기 쉽도록 하나의 메서드에 모아두는 편입니다.